services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050348  | Automatic failover to healthy backends                | Detect and redirect traffic away from unhealthy or failed backend servers to healthy ones automatically.                                                                                                                                                                                                                      |
+| CCC-050349  | Configurable standby backends                         | Configure backend targets that are only routed to if primary targets become unhealthy.                                                                                                                                                                                                                                        |
+| CCC-050350  | Connection draining on failover and failback          | Allows existing TCP sessions to remain active for a configurable time period even after backend VMs become unhealthy.                                                                                                                                                                                                         |
+| CCC-050351  | Health Checks                                         | Can perform health checks by sending HTTP, HTTPS, TCP, or SSL requests to specified endpoints on backend instances and evaluate the responses to determine the health status.                                                                                                                                                 |
+| CCC-050352  | Security Groups                                       | Can control inbound and outbound traffic based on source and destination IP addresses, port numbers, and protocols.                                                                                                                                                                                                           |
+| CCC-050353  | SSL Offloading / Termination                          | Can terminate SSL/TLS connections from clients and forwards decrypted traffic to backend servers.                                                                                                                                                                                                                             |
+| CCC-050354  | Auto-provisioned certificates                         | Can automatically generate, provision, and manage SSL/TLS certificates.                                                                                                                                                                                                                                                       |
```

Comment:
Can it really generate certificates automatically? In AWS, this is done through Certificate Manager and we only attach the SSL/TSL certificate to the load balancer so that it can accept https traffic.

Reply:
Yeah you're right. Auto-provisioning is handled by other services within each provider, but load balancing supports the use of them. Would you opt to remove this feature in this case?